packages/@aws-cdk/assertions/vendor-in.sh

```diff
@@ -17,6 +17,16 @@ echo "⏳ Vendoring in modules..."
 
 scriptdir=$(cd $(dirname $0) && pwd)
 cd $scriptdir
+
+if [[ $scriptdir != *"/packages/@aws-cdk/assertions"* ]]; then
```

Comment:
can we base this on an environment variable such as `SKIP_GEN` or such that will be enabled during the new phase we plan to introduce?

Reply:
Thought about this a bit more. Given that we decided to add a new phase that runs after the 'build' phase, the env var should set `XXX_PHASE=true` or something like that. We should decide what to call this phase.

Reply:
Done.